common/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -241,8 +242,8 @@ enum TaskStatus {
 message LoadFileRequest {
   repeated UfsStatus ufs_status = 1;
   required UfsReadOptions options = 2;
-  optional bool load_metadata_only = 3; // If set false, only the metadata of file will be loaded.
```

Comment:
why is this removed... i think zhihu will need to use this

Reply:
now we seperate the loadmetadata & load data task so if we specify ufsStatus and don't specify blocks then we only load metadata so this flag is no longer useful.